pxtlib/main.ts

```diff
@@ -162,7 +162,7 @@ namespace pxt {
     // this is set by compileServiceVariant in pxt.json
     export function setAppTargetVariant(variant: string): void {
         pxt.debug(`app variant: ${variant}`);
-        if (appTargetVariant === variant) return;
+        if (!!appTargetVariant === !!variant) return;
```

Comment:
I don’t think that’s right. For example an app variant might be “foobar”

Reply:
Oh of course, duh. Instead:
```suggestion
        if (appTargetVariant === variant || (!appTargetVariant && !variant)) return;
```